README.md

```diff
@@ -2,57 +2,159 @@
 
 ## HTTP
 
-### 1.  Building server application
+### 1.  Add frontend
 
-Copy production build of frontend application ([link](https://github.com/opendata-mvcr/dashboard-indexer-frontend)) to *frontend/build*   
-Build server aplication:
+Copy production build of frontend application ([link](https://github.com/opendata-mvcr/dashboard-indexer-frontend)) to *frontend/build*
 
-	 mvn clean install  
+### 2. Create .env file
 
-### 2. Create docker containers
+Create .env file in root directory of the project (same directory as "*docker-compose.yml*"). Insert folowing:
 
-Change settings in *docker-compose.yml* file. Then create images and start containers with command:
+	# Name of compose cluster of containers  
+	COMPOSE_PROJECT_NAME=**cluster-name**
+	
+	# Ports  
+	ELASTICSEARCH_PORT=**9201**
```

Comment:
Would add that there are sensible defaults for these vars (so no need to create .env props record for many of them)